Content.Server/Paper/PaperComponent.cs

```diff
@@ -10,6 +10,9 @@ public sealed class PaperComponent : SharedPaperComponent
         [DataField("content")]
         public string Content { get; set; } = "";
 
+        [DataField("localizedContent")]
+        public string LocContent = "";
+
```

Comment:
Datadefs shouldn't have copied string fields.

Reply:
> Datadefs shouldn't have copied string fields.

Should I make it look like Content filed?

Reply:
There should only be 1 field for string contents.

Reply:
@metalgearsloth Please explain the problem. Is Content field approach required?

Reply:
You shouldn't have 2 serialized string fields to store the 1 piece of data.